pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -394,15 +394,13 @@ type TestResults struct {
 	// Is the connection healthy
 	Success bool `json:"success"`
 
-	// Possible message
-	Message string `json:"message,omitempty"`
-
 	// Field related errors
-	Fields map[string]FieldError `json:"fields,omitempty"`
+	Errors []ErrorDetails `json:"errors,omitempty"`
```

Comment:
Does this apply on to the test repository endpoint? Just realised it'll break a lot of the app notifications as those expect errors to be an array of strings, but for the test endpoint we can adjust that.

Reply:
only changes the response to /test